Models/Board.cs

```diff
@@ -8,5 +8,9 @@ namespace LUDO.Models
 {
     internal class Board
     {
+        public List<Cell> GameCells { get; set; }
```

Comment:
Are we working on a single List solution instead of several arrays that we initially discussed (see umlet)? Just as long as it is simpler and it works :)